ui/lib/core/addon/components/ttl-picker2.js

```diff
@@ -74,7 +74,14 @@ export default TtlForm.extend({
 
     if (typeOf(value) === 'number') {
       // if the passed value is a number, assume unit is seconds
```

Comment:
does this comment still make sense? Because we're not assuming that it's seconds anymore, correct?

Reply:
We're still assuming that the number passed is referring to the number of seconds, not a number of minutes for example. I can add another comment explaining what the next chunk is doing though